### ISSUE TITLE
Make it easier to increase the size of text elements with the pinch gesture

### ISSUE THREAD

**Issue by megsfulton** [Automattic/portkey-android#532]
In the current build, it's challenging to scale up the text using a pinch gesture. It seems like you have to be very precise. 

@alexforcier - I know we discussed possibly having the slider during text editing which will help but I think we should address the gesture issue as well.

**Comment by megsfulton** [comment]
>Think I said it in a previous build. I still find selecting my text layer tricky. Feel like the tap target should be larger. Especially when there’s just one layer

^ this might also be related?

**Comment by aforcier** [comment]
> I know we discussed possibly having the slider during text editing which will help but I think we should address the gesture issue as well.

Yeah it feels more necessary now - I'll try and cobble together a first pass this week.

**Comment by aforcier** [comment]
The situation is improved with #561 - still keeping this open but pushing it down the priority list a bit.
